assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -145,59 +176,131 @@ export default function ModuleRecoveryAlert() {
+								'google-site-kit'
+							),
+							recoverableModules[ recoverableModulesList[ 0 ] ]
+								.name
+						),
+						{
+							a: (
+								<Link
+									href={ documentationURL }
+									external
+									aria-label={ __(
+										'Learn more',
+										'google-site-kit'
+									) }
+								/>
+							),
+						}
+					) }
+				</p>
+			);
+			actions = (
+				<CTALink
+					id={ id }
+					ctaLabel={ __( 'Remind me later', 'google-site-kit' ) }
+					onCTAClick={ () =>
+						dismissNotification( id, {
+							expiresInSeconds: DAY_IN_SECONDS,
+						} )
+					}
+				/>
 			);
 		} else {
-			description = __(
-				'The data for the following modules was previously shared with other users on the site by another admin who no longer has access. To restore access, the module must be recovered by another admin who has access.',
-				'google-site-kit'
+			description = (
+				<p className="googlesitekit-publisher-win__desc">
+					{ createInterpolateElement(
+						__(
+							'The data for the following modules was previously shared with other users on the site by another admin who no longer has access. To restore access, the module must be recovered by another admin who has access. <a>Learn more</a>',
+							'google-site-kit'
+						),
+						{
+							a: (
+								<Link
+									href={ documentationURL }
+									external
+									aria-label={ __(
+										'Learn more',
+										'google-site-kit'
+									) }
+								/>
+							),
+						}
+					) }
+				</p>
 			);
-			children = (
-				<ul className="mdc-list mdc-list--non-interactive">
-					{ recoverableModulesList.map( ( slug ) => (
-						<li className="mdc-list-item" key={ slug }>
-							<span className="mdc-list-item__text">
-								{ recoverableModules[ slug ].name }
-							</span>
-						</li>
-					) ) }
-				</ul>
+			actions = (
+				<Fragment>
+					<ul className="mdc-list mdc-list--non-interactive">
+						{ recoverableModulesList.map( ( slug ) => (
+							<li className="mdc-list-item" key={ slug }>
+								<span className="mdc-list-item__text">
+									{ recoverableModules[ slug ].name }
+								</span>
+							</li>
+						) ) }
+					</ul>
+					<CTALink
+						id={ id }
+						ctaLabel={ __( 'Remind me later', 'google-site-kit' ) }
+						onCTAClick={ () =>
+							dismissNotification( id, {
+								expiresInSeconds: DAY_IN_SECONDS,
+							} )
+						}
+					/>
+				</Fragment>
 			);
 		}
-	} else if ( userAccessibleModules.length === 1 ) {
-		description = sprintf(
-			/* translators: %s: module name. */
-			__(
-				'%s data was previously shared with other users on the site by another admin who no longer has access. To restore access, you may recover the module as the new owner.',
-				'google-site-kit'
-			),
-			recoverableModules[ userAccessibleModules[ 0 ] ].name
+	} else if ( userAccessibleModulesList.length === 1 ) {
+		description = (
+			<p className="googlesitekit-publisher-win__desc">
```

Comment:
Should we not use `assets/js/googlesitekit/notifications/components/common/Description.js` here? We shouldn't really be using `googlesitekit-publisher-win` classes in new components but it's okay to be part of the base components like this which can updated all at once.

Reply:
Ideally, we should use `Description` here. Even though these new components use the old classes, we will be redesigning the "internals" of all banners and setup CTAs in BNR3. So we will get rid of these classes completely when we create new layouts or refactor existing ones to use new consolidated CSS.